packages/gitbeaker-browser/src/KyRequester.ts

```diff
@@ -53,18 +54,26 @@ export function processBody(response) {
 }
 
 export async function handler(endpoint, options) {
+  const obeyRateLimit = true;
```

Comment:
Similar with ky https://github.com/sindresorhus/ky/#retry

Reply:
I think that doesn't work, because GitLab doesn't actually send any headers with `Retry-Later`. That's why it doesn't work